test/engine/double.test.lua

```diff
@@ -0,0 +1,13 @@
+env = require('test_run')
+test_run = env.new()
+engine = test_run:get_cfg('engine')
+
+test_space_with_double_field = box.schema.space.create('space_with_double', {
+    engine = engine,
+    format = { { 'id', 'unsigned' }, { 'double_field', 'double' } }
+})
+
+test_space_with_double_field:create_index('id', { parts = { 'id' } })
+
+box.space.space_with_double:insert { 1, 1 }
```

Comment:
I'd use `1.0` also. (It's the same `1` though once kept to Lua number, but regardless)